nmma/em/likelihood.py

```diff
@@ -67,20 +71,18 @@ def __init__(
         self.trigger_time = trigger_time
         self.tmin = tmin
         self.tmax = tmax
-        if isinstance(error_budget, (int, float, complex)) and not isinstance(
-            error_budget, bool
-        ):
+        self.dt = dt
+        if isinstance(error_budget, (int, float, complex)) and not isinstance(error_budget, bool):
             self.error_budget = dict(zip(filters, [error_budget] * len(filters)))
         elif isinstance(error_budget, dict):
             for filt in self.filters:
                 if filt not in error_budget:
                     raise ValueError(f"filter {filt} missing from error_budget")
             self.error_budget = error_budget
 
-        processedData = utils.dataProcess(
-            light_curve_data, self.filters, self.trigger_time, self.tmin, self.tmax
-        )
+        processedData = utils.dataProcess(light_curve_data, self.filters, self.trigger_time, self.tmin, self.tmax)
```

Comment:
does this still respect the character limit?

Reply:
I changed my local formatting length so maybe that re-formatted this as well. apart from linting. code will work fine